test/ConnectionManager.tests.js

```diff
@@ -88,6 +88,7 @@ define(function (require) {
                 expect(mockConnectionFactory.createConnection).toHaveBeenCalled();
                 expect(mockConnection.execute).toHaveBeenCalledWith(
                     jasmine.any(Request),
+                    jasmine.any(Object),
```

Comment:
actually we don't expect *any object* but an empty object (`{}` instead of `jasmine.any(Object)`).
Also, there's no test where request is called with its new optional parameter, so I guess the coverage is incomplete.